diff -urN eiskaltdcpp-2.2.7/dcpp/CMakeLists.txt eiskaltdcpp-2.2.7-patched/dcpp/CMakeLists.txt
--- eiskaltdcpp-2.2.7/dcpp/CMakeLists.txt	2012-05-30 05:55:25.000000000 +1100
+++ eiskaltdcpp-2.2.7-patched/dcpp/CMakeLists.txt	2012-10-13 02:12:29.160051391 +1100
@@ -81,7 +81,7 @@
 endif (WIN32)
 
 target_link_libraries (dcpp ${DHT_LIB} ${PTHREADS} ${BZIP2_LIBRARIES} ${ZLIB_LIBRARIES}
-${OPENSSL_LIBRARIES} ${GETTEXT_LIBRARIES} ${ICONV_LIBRARIES} ${WIN32_LIBS} ${APPLE_LIBS} ${LUA_LIBRARIES} ${UPNP} ${PCRE} ${IDNA_LIBRARIES} ${XATTR_LIBRARIES} ${HAIKU_LIB})
+${OPENSSL_LIBRARIES} ${GETTEXT_LIBRARIES} ${ICONV_LIBRARIES} ${WIN32_LIBS} ${APPLE_LIBS} ${LUA_LIBRARIES} ${UPNP} ${PCRE} ${IDNA_LIBRARIES} ${XATTR_LIBRARIES} ${HAIKU_LIB} -lboost_system)
 set_target_properties(dcpp PROPERTIES VERSION ${SOVERSION} OUTPUT_NAME "eiskaltdcpp")
 
 if (APPLE)
